src/test/java/org/mdbenefits/app/utils/TestUserFileUtilities.java

```diff
@@ -0,0 +1,18 @@
+package org.mdbenefits.app.utils;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+public class TestUserFileUtilities {
+    @CsvSource(delimiter=';', value = {
+        // passed in value; expected value
+        ".jpg, .jpeg, .csv,.pdf;.jpg, .jpeg, .csv, and .pdf",
```

Comment:
Feel silly mentioning it, but as we are doing joins and Oxford commas, I would add cases for 1 file type and 2 file types.

Reply:
Oh, that's a very good point.

Reply:
I added code to address this.  I think it was good to tighten this up some.